File: app/src/main/java/com/example/screen_confirm/MainActivity.java
package com.example.screen_confirm;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    public CardView cardView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        cardView=findViewById(R.id.card_confirm);
        cardView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(getApplicationContext(),"Action Confimed",Toast.LENGTH_LONG).show();
                Toast.makeText(getApplicationContext(),"Click on first card to move to the login page",Toast.LENGTH_LONG).show();
                Intent intent=new Intent(MainActivity.this,new_creator.class);
                startActivity(intent);



            }
        });
    }
}

File: app/src/main/java/com/example/huellay/MainActivity.kt
package com.example.huellay

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private lateinit var confirmButton: Button
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        confirmButton = findViewById(R.id.card_confirm)
        confirmButton.setOnClickListener(View.OnClickListener {
            Toast.makeText(this, "Action Confirmed", Toast.LENGTH_LONG).show()
            Toast.makeText(
                this,
                "Click on first card to move to the login page",
                Toast.LENGTH_LONG
            ).show()
            val intent = Intent(this, NewCreatorActivity::class.java)
            startActivity(intent)
        })
    }
}